[FIX] Header stock.picking fiscal fields.

mileo opened the pull request:
If you create a sale order to customer Contribuinte PE, invoice after delivery:
- Insert on item one: Mouse;
- Insert on item two: Ipad;

If you create a sale order to customer Contribuinte PE,  invoice after delivery:
- Insert on item one: Ipad;
- Insert on item two: Mouse;

And if you compare header fiscal fields they will be different.

rvalyi commented on l10n_br_sale_stock/models/stock.py:
```diff
@@ -0,0 +1,36 @@
+    def action_confirm(self):
+        """
+            Pass the department_id and the dispatch_note to the picking)
+        """
```
looks like the docstring doesn't match what the code does, better remove it then.

mileo committed:
[FIX] Header stock.picking fiscal fields.

If you create a sale order to customer Contribuinte PE, invoice after delivery:

- Insert on item one: Mouse;
- Insert on item two: Ipad;

If you create a sale order to customer Contribuinte PE,  invoice after delivery:

- Insert on item one: Ipad;
- Insert on item two: Mouse;

And if you compare header fiscal fields they will be different.

mileo commented:
@rvalyi fixed

mileo commented:
@lfdivino Poderia revisar este bug, ele é super importante
